### ISSUE TITLE
Input Focus

### ISSUE THREAD

**Issue by SereiaMaster** [Rocketseat/unform#146]
I'm not getting input focus
I'm trying to implement this hook that is in the official React documentation.

```js
function TextInputWithFocusButton() {
  const inputEl = useRef(null);
  const onButtonClick = () => {
    // `current` points to the mounted text input element
    inputEl.current.focus();
  };
  return (
    <>
      <input ref={inputEl} type="text" />
      <button onClick={onButtonClick}>Focus the input</button>
    </>
  );
}
```

**Comment by emilioheinz** [comment]
Please format your code using 

![image](https://user-images.githubusercontent.com/47385105/66592801-28804100-eb6c-11e9-8012-440d7c4f8029.png)

I'm not able to edit your issue, sorry.

**Comment by SereiaMaster** [comment]
> Please format your code using
> 
> ![image](https://user-images.githubusercontent.com/47385105/66592801-28804100-eb6c-11e9-8012-440d7c4f8029.png)
> 
> I'm not able to edit your issue, sorry.

Updated.
It's the React Documentation code ... I put the ref on Input inside the Form component

**Comment by diego3g** [comment]
Hello @SereiaMaster, you good?

We've released Unform 2.0 🎉

Now you can access the input reference by using the `getFieldRef` function. 

Check this guide to upgrade to v2: https://unform.dev/migration-guide

And this guide to understand how to focus the inputs: https://unform.dev/recipes/accessing-field-ref

I'll close the issue for now but feel free to reopen or create new ones if you have any problems.
